# Service Type for ShipBot arm operations
# dev_type: Type of device being manipulated
int8 dev_type
# dev_id: Used to encode which component of a device (for breaker)
int8 dev_id
# station_id: For special cases for corner
int8 station_id
---
# dev_ori: Orientation of device rotation plane (if applicable)
int8 dev_ori
# dev_pos: Specifically for shuttlecock
int8 dev_pos
# pose: Pose of device frame to reach to
geometry_msgs/Pose pose
